lib/rubocop/ast/node/mixin/conditional_node.rb

```diff
@@ -14,14 +14,6 @@ def single_line_condition?
         loc.keyword.line == condition.source_range.line
       end
 
-      # Checks whether the condition of the node is written on more than
-      # one line.
-      #
-      # @return [Boolean] whether the condition is on more than one line
-      def multiline_condition?
```

Comment:
This is public API? Do we know its truly unused?

Reply:
checked in the whole project and rubocop-rails and performance and it is not being used. More than happy to revert if you deem it dangerous to remove like this

Reply:
Well, technically speaking we just did 2 breaking changes. Maybe no one's really using this, but we still have to be careful with such types of changes.

Reply:
Ah, I only saw repos of rubocop-hq org about using these APIs. Certainly, these APIs are published interfaces, so this change is a breaking change. I was short of attention 💦